src/config.ts

```diff
@@ -186,23 +186,18 @@ function validateMaxItemsInChart(max: number | null) {
     }
 }
 
-function validateGroupBy(groupBy: string | undefined): string[] {
-    const defaultValue: string[] = ['os'];
+function validateGroupBy(groupBy?: string): string[] {
     if (groupBy === undefined) {
-        return defaultValue;
+        return ['os'];
     }
 
-    let arr: string[];
-    try {
-        arr = JSON.parse(groupBy.replaceAll("'", '"'));
-    } catch (e) {
-        // default
-        return defaultValue;
-    }
-    if (!Array.isArray(arr)) {
-        return defaultValue;
+    const keys = groupBy.split(',');
+
+    if (keys.length === 0) {
```

Comment:
I just tried `"".split(",")` and it returned `[""]`, so I think this should be

```suggestion
    if (keys.length === 1 && keys[0] === "") {
```